sample_apps/simple_app_under_keystone/instructions.md

```diff
@@ -8,12 +8,26 @@ No dependencies on anything, can build on any architecture and with any compiler
 
 ## Example App
 
+Start Qemu
+
```

Comment:
Can we briefly overview what we're gonna do?
e.g., we're going to use Keystone's docker image to emulate RISC-V machine with Keystone installed.

Reply:
resolved